connection.go

```diff
@@ -93,12 +93,13 @@ func (p *Pool) ExecQuery(query string) ([]byte, error) {
 }
 
 func (p *Pool) Exec(req *Request) ([]byte, error) {
-	conn := p.Get()
 	requestMessage, err := GraphSONSerializer(req)
 	if err != nil {
 		p.logger.Log("serializeRequestErr", err)
 		return nil, err
 	}
+	conn := p.Get()
+	defer p.Put(conn)
```

Comment:
While we are here let us improve the error messages as well. Could we log the query and params and clean up the error log to be in the usual format.

Did we check kibana for the error log, we should be able to match up the counts as well.